packages/edition-slices/__tests__/shared-tiles.base.js

```diff
@@ -35,9 +39,12 @@ jest.mock("@times-components/link", () => "Link");
 jest.mock("@times-components/gradient", () => "Gradient");
 
 const tile = mockEditionSlice(1).items[0];
+const dailyRegisterItem = mockDailyRegisterSlice().birthdaysToday;
 
-const testTile = Tile => {
-  const output = TestRenderer.create(<Tile onPress={() => {}} tile={tile} />);
+const testTile = (Tile, registerItem) => {
```

Comment:
This is more or less what I had in mind, so it'll allow us to use a different mock on certain tiles if needed as well:
```suggestion
const testTile = (Tile, tile = mockTile) => {
```

But this is not a blocking comment, it works either way :+1: